lib/find.coffee

```diff
@@ -117,6 +117,20 @@ module.exports =
       'find-and-replace:select-skip': (event) ->
         selectNextObjectForEditorElement(this).skipCurrentSelection()
 
+    maybeRegisterToAutocompletePlus = (atomPackage) =>
```

Comment:
It would be better to expose a service from autocomplete plus that exposes the ability to observe providers and consume it in this package.

But another question... what do you think about hard-coding support for find-and-replace editors directly in the default autocomplete provider rather than adding it dynamically? Then you'd only need to register the editor in this package.